proto/cosmos/protocolpool/v1/genesis.proto

```diff
@@ -4,11 +4,19 @@ package cosmos.protocolpool.v1;
 option go_package = "cosmossdk.io/x/protocolpool/types";
 
 import "cosmos/protocolpool/v1/types.proto";
+import "gogoproto/gogo.proto";
+import "cosmos_proto/cosmos.proto";
 
 // GenesisState defines the protocolpool module's genesis state.
 message GenesisState {
   // ContinuousFund defines the continuous funds at genesis.
   repeated ContinuousFund continuous_fund = 1;
   // Budget defines the budget proposals at genesis.
   repeated Budget budget = 2;
+
+  string to_distirbute = 3 [
```

Comment:
the other option is we run the distribution step as part of export, that would empty the kv store, i dont think there is an issue with either step. This was simpler